.eslintrc.json

```diff
@@ -157,6 +157,37 @@
       "rules": {
         "filenames-simple/naming-convention": "off"
       }
+    },
+    {
+      "files": ["**/api/src/**/*.ts"],
+      "plugins": ["eslint-plugin-jsdoc"],
+      "extends": ["plugin:jsdoc/recommended"],
+      "rules": {
+        "jsdoc/require-jsdoc": [
+          "warn",
+          {
+            "require": {
+              "ArrowFunctionExpression": true,
+              "ClassDeclaration": true,
+              "ClassExpression": true,
+              "FunctionDeclaration": true,
+              "FunctionExpression": true,
+              "MethodDefinition": true
+            },
+            "publicOnly": true
+          }
+        ],
+        "jsdoc/require-description-complete-sentence": "warn",
+        "jsdoc/no-undefined-types": "off",
```

Comment:
```suggestion
```
Since the TS recommended rules don't allow types, I don't think this is necessary.